gradle.properties

```diff
@@ -1,11 +1,11 @@
 # Kotlin
 version=1.2.0-alpha-2-SNAPSHOT
 group=org.jetbrains.kotlinx
-kotlin_version=1.3.21
+kotlin_version=1.3.30
```

Comment:
Need to search/replace version across all files (READMEs, example projects, etc)